File: crates/deslop-mcp/tests/fixtures/dart-mcp/cart_totals.dart
double computeCartTotal(List<LineItem> items, double taxRate, double discount) {
  var subtotal = 0.0;
  for (final item in items) {
    final lineTotal = item.unitPrice * item.quantity;
    if (item.onSale) {
      subtotal = subtotal + lineTotal * 0.9;
    } else {
      subtotal = subtotal + lineTotal;
    }
  }
  final taxed = subtotal * (1 + taxRate);
  final afterDiscount = taxed - discount;
  return afterDiscount < 0 ? 0 : afterDiscount;
}

File: crates/deslop-mcp/tests/fixtures/fsharp-mcp/cart_totals.fs
module CartTotals

let computeCartTotal (items: LineItem list) (taxRate: float) (discount: float) : float =
    let mutable subtotal = 0.0
    for item in items do
        let lineTotal = item.UnitPrice * item.Quantity
        if item.OnSale then
            subtotal <- subtotal + lineTotal * 0.9
        else
            subtotal <- subtotal + lineTotal
    let taxed = subtotal * (1.0 + taxRate)
    let afterDiscount = taxed - discount
    if afterDiscount < 0.0 then 0.0 else afterDiscount
